activation/activation.go

```diff
@@ -147,10 +146,9 @@ func (b *Builder) loop() {
 
 // PublishActivationTx attempts to publish an atx for the given epoch, it returns an error if an atx cannot be created
 // publish atx may not produce an atx each time it is called, that is expected behaviour as well.
-func (b *Builder) PublishActivationTx(epoch types.EpochId) error {
+func (b *Builder) PublishActivationTx(epoch types.EpochId) (bool, error) {
```

Comment:
why add the boolean, couldn't fund where this value is used in production code

Reply:
it's not, I've added it so I can test this case where the ATX isn't published not due to an error.